test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -277,13 +327,14 @@ public void LoadingBuiltInZTPackageAddsItToLibrary()
                 Preferences = CurrentDynamoModel.PreferenceSettings,
 
             };
+
+            CurrentDynamoModel.SearchModel.ClearAllEntries();
```

Comment:
the fact that you need to do this has me a bit worried about loading ZT nodes multiple times into search- a bug that keeps coming back - can you explain it?

Reply:
The problem seems to be only during tests (at least as far as I can tell)
Many tests use new instances of `PackageLoader`...expecting that there will be a clean slate every time.
In other words...many tests load packages into the VM....and do not clean it up (not even possible without a restart ...I think).
I this case...some tests (that ran before the `LoadingBuiltInZTPackageAddsItToLibrary` had loaded a bunch of packages). Now when the current test checks the SearchModel...it actually sees packages from previous tests, that have not been cleaned up.
My proposal, using `CurrentDynamoModel.SearchModel.ClearAllEntries`, is just a band aid for the immediate issue.

let me know if you can think of a better way of doing it ...or if you think this (or something similar)might happen when clients use Dynamo.

Reply:
OK..I forgot that the DynamoModel is torn down after every test.
I updated the code with a proper fix